internal/api/front/proxy/prometheus/endpoint.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Amadeus s.a.s
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheus
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+)
+
+type Endpoint struct {
+	prometheusUrls []*url.URL
+}
+
+func NewEndpoint(urls []string) (*Endpoint, error) {
+	var prometheusUrls []*url.URL
+	for _, u := range urls {
+		if prometheusUrl, err := url.Parse(u); err != nil {
+			return nil, err
+		} else {
+			prometheusUrls = append(prometheusUrls, prometheusUrl)
+		}
+	}
+	return &Endpoint{
+		prometheusUrls: prometheusUrls,
+	}, nil
+}
+
+func (e *Endpoint) RegisterRoutes(g *echo.Group) {
```

Comment:
At some point we will have to tackle how to connect the promql editor with a remote Prometheus. But I'm filling since we are storing the datasource with the Prometheus URL, we can do something better than that.

So I think, we should think a bit more about this topic before going furthermore. 
So to avoid to block this PR, I propose to remove this Proxy thinguy and to create an issue to talk about this topic.

Reply:
removed proxy